src/angular/styles/_typography.scss

```diff
@@ -625,6 +630,10 @@
       --sbb-icon-sidebar-background-color: var(--sbb-color-black);
       --sbb-icon-sidebar-background-color-active: var(--sbb-color-charcoal);
       --sbb-icon-sidebar-collapse-button-background-color: var(--sbb-color-background);
+
+      // Pagination
```

Comment:
Nit-pick: Please order the component names alphabetically. `// Pagination` should be before `// Sidebar`.

There are some other component names which are not ordered. We'll change it later (not in this PR to avoid merge conflicts).